lib/middleman-prismic.rb

```diff
@@ -1,47 +1,42 @@
 require 'prismic'
+require 'middleman-core'
 require 'middleman-prismic/version'
 require 'middleman-prismic/commands/prismic'
 
-module MiddlemanPrismic
-  class << self
-    attr_reader :options
-  end
-
-  class Core < Middleman::Extension
-
-    option :api_url, nil, 'The prismic api url'
-    option :release, 'master', 'Content release'
-    option(
-      :link_resolver,
-      ->(link) {"http://www.example.com/#{link.type.pluralize}/#{link.slug}"},
-      'The link resolver'
-    )
-    option :custom_queries, {}, 'Custom queries'
+module Middleman
+  module Prismic
+    class << self
+      attr_reader :options
+    end
 
-    def initialize(app, options_hash={}, &block)
-      super
+    class Core < ::Middleman::Extension
+      option :api_url, nil, 'The prismic api url'
+      option :release, 'master', 'Content release'
+      option(
+        :link_resolver,
+        ->(link) {"http://www.example.com/#{link.type.pluralize}/#{link.slug}"},
+        'The link resolver'
```

Comment:
This seems to be help text, but I'm not sure what it means. Is there a more descriptive thing that we can use here?

Reply:
Yeah, I'm not totally sure what this is or whether it's required.